src/coreclr/jit/target.h

```diff
@@ -305,56 +364,69 @@ struct regMaskTP
 
     SingleTypeRegSet GetPredicateRegSet() const
     {
+#ifdef HAS_MORE_THAN_64_REGISTERS
+        return getHigh();
+#else
         return getLow();
```

Comment:
Shouldn't it be `getLow() >> REG_P0` or something like that?

Reply:
For x64, when we create the `RBM_ALLMASK`, we already have bits set to 0 except the ones that represent mask registers, hence should be safe to just return `getLow()`.